frontend/src/queries/nodes/InsightViz/InsightVizDisplay.tsx

```diff
@@ -176,7 +176,7 @@ export function InsightVizDisplay({
             return (
                 <>
                     {exportContext && (
-                        <div className="flex items-center justify-between my-4 mx-0">
+                        <div className="flex items-center justify-between my-4 mx-0 detailed-results-header-panel">
```

Comment:
Unfortunately there's no other way to target this since both this div and the div below belong to `InsightVizDisplay`, and I'm not happy with the margin used there.

Initially I didn't want to touch the global styling here since it's not our component, but I'm gonna go ahead and do it, because it's a tiny change and does make sense in my opinion. Screenshot ⬇️

Reply:
The change: the margin-top for the h2 should be more than the margin-bottom, since the h2 describes the component below.

|Before|After|
|----|----|
|<img width="304" alt="image" src="https://github.com/PostHog/posthog/assets/22996112/ba240404-cadc-4545-9213-cabc314b0b17">|<img width="300" alt="image" src="https://github.com/PostHog/posthog/assets/22996112/bc7a22bc-88b9-410a-b57e-4519dec2afb6">|

Reply:
That sounds reasonable, not sure what analytics prefers: @thmsobrmlr @Twixes would you prefer this be a configuration option that adds the class, and is default off, or the current code?

Orrr, would you be okay with making this the default everywhere?

Reply:
I agree with the general reasoning that headings should be close to the things they describe. In this instance that doesn't look good due to the export button on the right side. Thus, no to default everywhere. 

Don't have a preference for special class or config option.

Reply:
oh good point, didn't realize about the export button 👍 

then I'll revert back to the original margins, we can live with it 😄